### ISSUE TITLE
Showing theme widget locations inside an editable Elementor page/post

### ISSUE THREAD

**Issue by paaljoachim** [pojome/elementor#306]
**Genesis child themes rely a lot on widget locations.**

For instance Altitude Pro is a parallax/one page popular theme from Genesis.

It contains these default widget locations:
![screen shot 2016-07-23 at 10 07 10](https://cloud.githubusercontent.com/assets/5323259/17076835/4bbace80-50bd-11e6-8cc0-0d7953dddd88.png)

Add a widget to each of the Front Page widget locations and these Customizer featured sections images show up:

![screen shot 2016-07-23 at 10 08 24](https://cloud.githubusercontent.com/assets/5323259/17076873/11c8bf60-50be-11e6-81e0-3aa9448e7a6f.png)

**Using Elementor with this one page theme from Genesis:**

I added an empty text edit widget to the widget locations to be able to use the customizer featured section images. Then went to edit the front page with Elementor.

![screen shot 2016-07-23 at 10 14 48](https://cloud.githubusercontent.com/assets/5323259/17076883/4f6a833a-50be-11e6-8eba-766761af95f1.png)

Since the front page uses widgets the_content is not used and it creates the above message.

That means that Altitude Pro from Genesis is not a good theme to use with Elementor (right now) because it relies on having widgets in the front page widget locations and adjusting the front page featured sections that will become visible when there is a widget present in the corresponding widget location.

Meaning add a widget to front page 1 widget location and add an image in the customizer to front page 1 featured section. See the image show up in the background with the widget information on top.

I have a Github repo for my custom Altitude Pro theme:
https://github.com/paaljoachim/custom-altitude-pro

Bottom line is:
Can we have widget locations show up inside the editable Frontend Elementor page/post? Having the locations show up one can drag Elements/widgets into them making it possible to use themes such as Altitude Pro with Elementor as well as other Genesis themes that have a similar setup of widget locations and customizer sections.

Associated issue:
https://github.com/pojome/elementor/issues/64

**Comment by arielk** [comment]
Sorry, but we're working with a different method, you can read more about this here: https://docs.elementor.com/article/56-content-area-not-found
